src/sentry/quotas/base.py

```diff
@@ -401,6 +401,13 @@ def get_event_retention(self, organization):
         """
         return _limit_from_settings(options.get("system.event-retention-days"))
 
+    def get_downsampled_event_retention(self, organization):
+        """
+        Returns the retention for downsampled events in the given organization in days.
+        Returns ``None`` if downsampled events are to be stored indefinitely.
+        """
+        return 0
```

Comment:
Comment says it returns `None`, this is actually `0`.

Reply:
No, we wouldn't. If you want a better default here, we can return `get_event_retention()`